homework-3/src/views/RestorePassword.js

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
 import { StylBoxMain } from '../components/common/Boxes/StylBoxMain';
-import FormRestorePassword from '../components/FormRestorePassword';
+import FormRestorePassword from '../components/Forms/FormRestorePassword';
 import BoxFormSignIn from './../components/common/Boxes/StylBoxFormSignIn';
+import Loader from './../components/Loader/Loader';
 
 const RestorePassword = ({ title }) => {
   React.useEffect(() => {
     document.title = title || '';
   }, [title]);
 
-  return (
-    <>
-      <StylBoxMain>
-        <BoxFormSignIn>
-          <FormRestorePassword />
-        </BoxFormSignIn>
-      </StylBoxMain>
-    </>
-  );
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
```

Comment:
I heard that setTimeout is not a good practice

Reply:
Removed